.env.template

```diff
@@ -0,0 +1,30 @@
+SMART_CONTRACT_PATH=./contracts/deployments/networks.json
+
+# Variables injected in the HTML page
+
+HTML_TITLE=teledisko dao
+
+# Open Graph Protocol properties. See https://ogp.me/
+HTML_DESCRIPTION=Teledisko dao
+HTML_BASE_URL=https://localhost:3000/
+HTML_IMAGE=https://localhost:3000/preview.png
+HTML_IMAGE_ALT=A hammer
+
+# Twitter properties. See:
+# https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup
+HTML_TWITTER_CARD=summary_large_image
+HTML_TWITTER_SITE=@vrde
```

Comment:
I guess we need to remove it or create a twitter handle, let's ignore it for nao

Reply:
if twitter will still exist... ⛵🪨

Reply:
![FhN4NoVWQAMQjIq](https://user-images.githubusercontent.com/134680/202745831-5f6fecc6-35a8-4256-b234-dd5924807b69.png)